Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -590,36 +606,54 @@ def format_to_iso(date_string):
     return date_string
 
 
-def fetch_incidents(proxies):
-    last_run = demisto.getLastRun()
-    last_fetch = last_run.get('time')
-
-    # handle first time fetch
-    if last_fetch is None:
-        last_fetch, _ = parse_date_range(date_range=FETCH_TIME, date_format='%Y-%m-%dT%H:%M:%S.%f', utc=False,
-                                         to_timestamp=False)
-        last_fetch = parse(str(last_fetch))
-        last_fetch_timestamp = int(last_fetch.timestamp() * 1000)
+def get_time_range(last_fetch: Union[str, None] = None, fetch_time=FETCH_TIME, time_range_start=TIME_RANGE_START,
+                   time_range_end=TIME_RANGE_END, time_field=TIME_FIELD) -> Dict:
+    """
+    Creates the time range filter's dictionary based on the last fetch and given params.
```

Comment:
Lets try to give more details on what is the logic we work by to create the time filter, which may help in debugging/solving bugs later on.